scripts/Inelastic/Direct/ReductionHelpers.py

```diff
@@ -228,10 +228,12 @@ def build_subst_dictionary(synonims_list=None) :
         if len(keys) < 2 :
             raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present")
         if len(keys[0]) == 0:
-            raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present, but the first key is empty")
+            raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present, "
```

Comment:
I counted and found that when I view the whole file in GIT The first 140 characters are shown from a line, but when displaying changes only 133 characters are shown.